modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -636,27 +740,25 @@ REGION_ID with the region to retrieve metrics from,
 STORE_ID with the store ID to retrieve metrics for, 
 and BEARER_TOKEN with your bearer token:
 
-[source,console,linenums]
+[source,bash,linenums]
 ----
  curl -X GET \
-  'https://object-store-stats.anypoint.mulesoft.com/api/v1/organizations/ORG_ID/environments/ENV_ID/regions/REGION_ID/stores/STORE_ID?startDate=2021-06-13T17%3A51%3A54.000Z&endDate=2021-09-13T17%3A51%3A54.000Z&period=1month' \
-  -H 'authorization: Bearer BEARER_TOKEN' 
+  'https://object-store-stats.anypoint.mulesoft.com/api/v1/organizations/<ORG_ID>/environments/<ENV_ID>/regions/<REGION_ID>/stores/<STORE_ID>?startDate=2024-08-13T17%3A51%3A54.000Z&endDate=2024-09-13T17%3A51%3A54.000Z&period=1month' \
+  -H 'authorization: Bearer <BEARER_TOKEN>' 
 ----
 
-To retrieve the STORE_ID, see <<Example: Retrieve a List of Object Stores>>.
+To retrieve the STORE_ID, see <<example-retrieve-list>>.
 
 This request returns a response similar to the following:
 
 [source,json,linenums]
 ----
 [
- {"timeStamp":"2021-06-01T00:00:00Z","objectStoreRequestCount":3},
- {"timeStamp":"2021-07-01T00:00:00Z","objectStoreRequestCount":30},
- {"timeStamp":"2021-08-01T00:00:00Z","objectStoreRequestCount":1},
- {"timeStamp":"2021-09-01T00:00:00Z","objectStoreRequestCount":25}
+ {"timeStamp":"2024-08-01T00:00:00Z","objectStoreRequestCount":1},
+ {"timeStamp":"2024-09-01T00:00:00Z","objectStoreRequestCount":25}
 ]
 ----
-
+[[download-osv2-apis]]
 == Download Object Store APIs from Exchange 
 
 To download the Object Store APIs from Exchange, click *Download* and select the format to download: RAML, OAS, or as a connector file for Mule 3 or Mule 4:
```

Comment:
To download the Object Store APIs from Exchange:

. Go to Exchange and click *Download*. 
. Select the format to download:
* RAML
* OAS
* Mule 4 or Mule 3 connector file